pipeline/bump-submodule/task

```diff
@@ -30,5 +31,6 @@ pushd "release-repo/$SUBMODULE_PATH"
   git clean -ffd
 popd
 
-git_commit_all release-repo "${COMMIT_MESSAGE:-Bump ${submodule_name} submodule}"
+git_commit_all release-repo "${COMMIT_MESSAGE:-Bump ${submodule_name} submodule to SHA: ${target_sha}
```

Comment:
I think the convention is for a blank line after the first (I think for email or some other obscure reason...).

I don't have strong feelings either way though.